components/datadog/agent/params.go

```diff
@@ -93,17 +121,16 @@ func parseVersion(s string) (os.AgentVersion, error) {
 			return version, fmt.Errorf("invalid version of the Agent: %v. The Agent version should starts with `7.` or `6.`", s)
 		}
 	}
+
+	// Best-effort attempt to detect betas / RCs, and redirect them to the staging/beta location.
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities. -->
**💭 thought**
Not related to this PR: it might be confusing, but also might never happen, that if someone tries to install rc.2 while only rc.1 is released, they would install the latest rc, thus rc.1